<html>

<head>
<meta http-equiv=Content-Type content="text/html; charset=us-ascii">
<meta name=Generator content="Microsoft Word 12 (filtered)">
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:Wingdings;
	panose-1:5 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:10.0pt;
	margin-left:0cm;
	line-height:115%;
	font-size:11.0pt;
	font-family:"Calibri","sans-serif";}
a:link, span.MsoHyperlink
	{color:blue;
	text-decoration:underline;}
a:visited, span.MsoHyperlinkFollowed
	{color:purple;
	text-decoration:underline;}
p.MsoListParagraph, li.MsoListParagraph, div.MsoListParagraph
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:10.0pt;
	margin-left:36.0pt;
	line-height:115%;
	font-size:11.0pt;
	font-family:"Calibri","sans-serif";}
p.MsoListParagraphCxSpFirst, li.MsoListParagraphCxSpFirst, div.MsoListParagraphCxSpFirst
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:36.0pt;
	margin-bottom:.0001pt;
	line-height:115%;
	font-size:11.0pt;
	font-family:"Calibri","sans-serif";}
p.MsoListParagraphCxSpMiddle, li.MsoListParagraphCxSpMiddle, div.MsoListParagraphCxSpMiddle
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:36.0pt;
	margin-bottom:.0001pt;
	line-height:115%;
	font-size:11.0pt;
	font-family:"Calibri","sans-serif";}
p.MsoListParagraphCxSpLast, li.MsoListParagraphCxSpLast, div.MsoListParagraphCxSpLast
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:10.0pt;
	margin-left:36.0pt;
	line-height:115%;
	font-size:11.0pt;
	font-family:"Calibri","sans-serif";}
.MsoPapDefault
	{margin-bottom:10.0pt;
	line-height:115%;}
@page Section1
	{size:595.3pt 841.9pt;
	margin:1.0cm 54.0pt 28.4pt 54.0pt;}
div.Section1
	{page:Section1;}
 /* List Definitions */
 ol
	{margin-bottom:0cm;}
ul
	{margin-bottom:0cm;}
-->
</style>

</head>

<body lang=RU link=blue vlink=purple>

<div class=Section1>

<p><b><span>&nbsp;</span></b></p>

<p><b><span>Introduction</span></b></p>

<p><span>The
major form of epigenetic information within the DNA molecule itself in
mammalian cells is DNA methylation, that is the covalent addition of a methyl
group to the 5-position of cytosine, mostly within the CpG dinucleotides in
somatic cells. DNA methylation is involved in the control of gene expression,
regulation of parental imprinting and stabilization of X chromosome
inactivation as well as maintenance of the genome integrity. DNA methylation is
mediated by a family of DNA methyltransferase enzymes (DNMTs). In mammals,
three DNMTs have been identified so far in the human genome, including the two
de novo methyltransferases (DNMT3A and DNMT3B) and the maintenance
methyltransferase (DNMT1), which is generally the most abundant and active of
the three. DNMT1 is responsible for duplicating patterns of DNA methylation
during replication and is essential for mammalian development and cancer cell
growth. Therefore, specific inhibition of DNA methylation is an attractive
approach for cancer therapy.</span></p>

<p><span>&nbsp;</span></p>

<p><b><span>Goal</span></b></p>

<p><span>We
provide a library of 4387 compounds which contains some validated active compounds
having IC<sub>50</sub> &lt; 10 &micro;M. You task is to select 100 compounds to
"submit" them for biological tests as DNMT1 inhibitors. The goal is to achieve
the highest hit rate. In order to achieve
this you can develop models based on available data and perform virtual
screening of the library to select 100 promising DNMT1 (</span><a
href="http://www.uniprot.org/uniprot/P26358"><span>http://www.uniprot.org/uniprot/P26358</span></a><span>)
inhibitors.</span></p>

<p><span>&nbsp;</span></p>

<p><b><span>Data</span></b></p>

<p><i><u><span>Compounds</span></u></i><i><span>:</span></i></p>

<p><a href="https://www.dropbox.com/s/0um980peo7jrjjg/train.sdf?dl=0">train.sdf</a> (<a href="https://www.dropbox.com/s/72j9toa0c06x1tk/train.smi?dl=0">train.smi</a>)</p>

<p class=MsoListParagraphCxSpFirst style='text-align:justify;text-indent:-18.0pt'><span
lang=EN-US >-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif"'>structure or canonical SMILES</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-align:justify;text-indent:-18.0pt'><span
lang=EN-US >-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif"'>Name &#8211; CHEMBLID</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-align:justify;text-indent:-18.0pt'><span
lang=EN-US >-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif"'>CANONICAL_SMILES &#8211; SMILES string</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-align:justify;text-indent:-18.0pt'><span
lang=EN-US >-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif"'>CMPD_CHEMBLID (the same as Name)</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-align:justify;text-indent:-18.0pt'><span
lang=EN-US >-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif"'>pIC50 &#8211; 1/log10(IC50), where IC50 is
in mol/l, (float)</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-align:justify;text-indent:-18.0pt'><span
lang=EN-US >-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif"'>IC50_class &#8211; 0 (pIC50 &lt;= 7) or
1 (pIC50 &gt; 7), (integer)</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-align:justify;text-indent:-18.0pt'><span
lang=EN-US >-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif"'>ASSAY_TYPE &#8211; ChEMBL assay type: B
&#8211; binding, F &#8211; functional, (string)</span></p>

<p class=MsoListParagraphCxSpLast style='text-align:justify;text-indent:-18.0pt'><span
lang=EN-US >-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif"'>DESCRIPTION &#8211; assay short description,
(string)</span></p>

<p><span>The
data set consists of 292 compounds taken from ChEMBL. Compounds were separated
on active and inactive ones. Actives are compounds inhibiting DNMT1 for more
than 50% at concentration lower than 10 &micro;M.</span></p>

<p><a href="https://www.dropbox.com/s/5fqixrt5jh1g8nf/train.ldb.zip?dl=0">train.ldb</a> (<a href="https://www.dropbox.com/s/wt8ib4rvjgcxcaj/train_active.ldb.zip?dl=0">train_active.ldb</a> and <a href="https://www.dropbox.com/s/9d2xwjat1daw1ka/train_inactive.ldb.zip?dl=0">train_inactive.ldb</a>)</p>

<p><span>LigandScout
binaries with up to 200 precomputed conformers and pharmacophores per
compound from the training set using icon-best method. Active/inactive files were generated
separately for compounds having DNMT1_class 0 and 1. Train.ldb contains all compounds of the training set.</span></p>

<p><a href="https://www.dropbox.com/s/hq2920e5uzcnuy4/train_PDBQT.zip?dl=0">train.pdbqt</a></p>

<p><span>The file suitable for docking with Autodock Vina</span></p>

<p><a href="https://www.dropbox.com/s/c0ooltf9tzmtare/blind.sdf?dl=0">blind.sdf</a> (<a href="https://www.dropbox.com/s/k7xcvq4mehhobvq/blind.smi?dl=0">blind.smi</a>)</p>

<p class=MsoListParagraphCxSpFirst style='text-align:justify;text-indent:-18.0pt'><span
lang=EN-US >-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif"'>structure or canonical SMILES</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-align:justify;text-indent:-18.0pt'><span
lang=EN-US >-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif"'>Name &#8211;name of a compound, (string)</span></p>

<p class=MsoListParagraphCxSpLast style='text-align:justify;text-indent:-18.0pt'><span
lang=EN-US >-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif"'>external_name &#8211; the same as Name,
(string)</span></p>

<p><span>The set of compounds which includes known actives, inactives and decoys (4350
inactive/decoys and 37 active compounds). This will be used as an external test
set.</span></p>

<p><a href="https://www.dropbox.com/s/1yty3kagctpnovh/blind.ldb.zip?dl=0">blind.ldb</a></p>

<p><span>LigandScout binary file with precomputed up to 200 conformers and pharmacophores per
compound from blind_final.sdf using icon-best method.</span></p>

<p><a href="https://www.dropbox.com/s/jah9kuiqyf9li3p/test_PDBQT.zip?dl=0">blind.pdbqt</a></p>

<p><span>The file suitable for Autodock Vina screening.</span></p>

<p><span>&nbsp;</span></p>

<p><i><u><span>Protein structures</span></u></i><i><span>.</span></i></p>

<p><span>Participants
can use data from Protein DataBank. The list of available structures for
proteins can be looked at uniprot.org (</span><a
href="http://www.uniprot.org/uniprot/P26358#structure"><span
>http://www.uniprot.org/uniprot/P26358#structure</span></a><span>).
There are complexes free proteins. 3SWR structure can be a good starting point
for structure-based modeling but it is not necessary.</span></p>

<p><span>&nbsp;</span></p>

<p><i><u><span>Additional data. </span></u></i></p>

<p><span>Participants
are allowed to use any public resources to collect additional data and use it
for modeling.</span></p>

<p><span>&nbsp;</span></p>

<p><b><span>Evaluation</span></b></p>

<p><span>Participants
should provide 100 selected compounds. Participants will be ranked according to
the score calculated as a hit rate for selected compounds relatively to the
maximum achievable hit rate. Thus the score will be within the range 0-1 (higher
is better). Up to 10 submissions will be allowed.</span></p>

<p><i><b><span>Score = hit rate for 100 selected compounds / max achievable hit rate</span></i></b></p>

<p><span>&nbsp;</span></p>

<p><b><span>Submissions</span></b></p>

<p><span>The
web application available at <a href="http://158.194.101.252:3838/challenge/">http://158.194.101.252:3838/challenge/</a>
should be used for submitting of selected IDs of compounds. You may provide the
list of IDs in a text file with one ID per line or you may paste IDs in the text
field (one ID per line). Filters are applied without pressing Refersh button.</span></p>

<p><span>&nbsp;</span></p>

<p><b><span>Tools</span></b></p>

<p><span>Participants
may use any <i>free</i> standalone or web-based chemoinformatic tools to
develop models and perform virtual screening. Using of commercial software is
prohibited if the license is not available for all participants. Currently only
LigandScout license will be made available for participants during the course.</span></p>

<p><span>&nbsp;</span></p>

<p><b><span>Additional info</span></b></p>

<p><span>Participants
can work alone or can form groups consisting of up to three people. The blind
set will be made available during the course.</span></p>

<p><span>&nbsp;</span></p>

<p><b><span>Prizes</span></b></p>

<p><span>Teams which possess first three places will be rewarded with somevaluable prizes. The only requirement they should make a short presentation describing their winner solution.</span></p>

<p><span>&nbsp;</span></p>

<p><b><span>Sponsors</span></b></p>

<p><span>The workshop and the competition are supported by</span></p>
<ul>
   <li>
     <p>LifeChemicals company - <a href="http://lifechemicals.com">lifechemicals.com</a></p>
     <p><img src="https://photos-5.dropbox.com/t/2/AACzDF5J34x5kgISbNKI16xhfgmiT8nLZuGQCsTX8jpJZA/12/125665113/png/32x32/1/_/1/2/LC_logo.png/ELmc5GAY3QIgAigC/PBgfJEyejVz_RrhZ8pUPknk-o6PiZDiiY6S4XnOC1dI?preserve_transparency=1&size=800x600&size_mode=3"/></p>
   </li>
   <li>ELEXIR project - <a href=""></a></li>
</ul>

<p><span>&nbsp;</span></p>

</div>

</body>

</html>
